utils/utils.js

```diff
@@ -631,24 +631,24 @@ const isDefined = (x) => x !== undefined;
 const isNotNull = (x) => x !== null;
 const isDefinedAndNotNull = (x) => isDefined(x) && isNotNull(x);
 
-const getDataFromSource = (src, dest, key, properties) => {
+const getDataFromSource = (src, dest, properties) => {
   const data = {};
   if (isArray(src)) {
     for (let index = 0; index < src.length; index += 1) {
-      if (src[index] === key.toLowerCase()) {
-        data[dest] = properties[key].toString();
+      if (properties[src[index]]) {
+        data[dest] = properties[src[index]]
```

Comment:
If any of the values in 'properties' object evaluate to false, they wouldn't be added to the return data now. Do you see any risk with that?

Reply:
Yes, thanks, this is the intended behavior